charts/fetcher/templates/bootstrap-mongodb.yaml

```
@@ -0,0 +1,118 @@
+{{- if .Values.global.externalMongoDefinitions.enabled }}
+{{- $creds := .Values.global.externalMongoDefinitions.fetcherCredentials -}}
+{{- $rolesJSON := toJson $creds.roles -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "fetcher.fullname" . }}-bootstrap-mongodb
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)